manifest.json

```
@@ -508,16 +530,5 @@
          "type" : "color",
          "version" : "0.1"
       },
```

```suggestion
      }
```
because JSON is picky.